test/recipe/admin/components/Baked/Menu.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <PanelMenu v-if="schema" :model="items" class="w-full md:w-80">
```

Comment:
`Menubar` would be better. Rich transient would be an item, and samples would be listed under that parent.